frontend/src/metabase/dashboard/actions/cards.js

```diff
@@ -71,6 +76,33 @@ export const addCardToDashboard =
     dispatch(loadMetadataForDashboard([dashcard]));
   };
 
+export const removeCardFromDashboard = createThunkAction(
+  REMOVE_CARD_FROM_DASH,
+  ({ dashcardId, cardId }) =>
+    (dispatch, _getState) => {
+      dispatch(cancelFetchCardData(cardId, dashcardId));
+      return { dashcardId, cardId };
```

Comment:
I don't see we change any reducers that use `REMOVE_CARD_FROM_DASH`, so based on the previous call
https://github.com/metabase/metabase/blob/5c1b2af3b14495a6199b23dda52667a15a701e8b/frontend/src/metabase/dashboard/components/DashboardGrid.jsx#L297-L299

I think we should just return `dashcardId` not also `cardId` to avoid confusion.